persistent/Database/Persist/Quasi/Internal.hs

```diff
@@ -497,10 +309,10 @@ data UnboundEntityDef
     -- the field?" yet, so we defer those to the Template Haskell execution.
     --
     -- @since 2.13.0.0
-    , unboundEntityDefSpan :: Maybe Span
+    , unboundEntityDefSourceSpan :: Maybe SourceSpan
```

Comment:
kind of same, let's keep function name the same so it's not a breaking change - we can break more gently with warnings/deprecations